2016/05/what-is-react.js-and-why-use-react.md

```diff
@@ -0,0 +1,41 @@
+I have been using [React.js](https://facebook.github.io/react/index.html), a JavaScrip library for creating user interfaces, for about 4 months now.  You might want to read the official [getting-started](https://facebook.github.io/react/docs/getting-started.html) or [Tutorial](https://facebook.github.io/react/docs/tutorial.html) pages.
```

Comment:
Maybe we should add some header information on each post, like Jekyll does. something like

````
---
title: Blogging Like a Hacker
---